modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java

```diff
@@ -310,40 +309,20 @@ private List<NameValuePair> getOIDCInitiationRequestParams() {
         return urlParameters;
     }
 
-    private ServiceProvider createServiceProvider() throws Exception {
-
-        ServiceProvider serviceProvider = new ServiceProvider();
-        serviceProvider.setApplicationName(OIDC_APP_NAME);
-        serviceProvider.setManagementApp(true);
-        applicationManagementServiceClient.createApplication(serviceProvider);
-        serviceProvider = applicationManagementServiceClient.getApplication(OIDC_APP_NAME);
-
-        InboundAuthenticationRequestConfig requestConfig = new InboundAuthenticationRequestConfig();
-        requestConfig.setInboundAuthKey(oidcAppClientId);
-        requestConfig.setInboundAuthType("oauth2");
-        if (StringUtils.isNotBlank(oidcAppClientSecret)) {
-            Property property = new Property();
-            property.setName("oauthConsumerSecret");
-            property.setValue(oidcAppClientSecret);
-            Property[] properties = {property};
-            requestConfig.setProperties(properties);
-        }
+    private void addAdminUser() throws Exception {
+        UserObject userInfo = new UserObject();
```

Comment:
Shall we add the missing new line?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096